benches/bigint.rs

```diff
@@ -255,3 +255,39 @@ fn pow_bench(b: &mut Bencher) {
         }
     });
 }
+
+#[bench]
+fn modpow(b: &mut Bencher) {
+    let mut rng = get_rng();
+    let base = rng.gen_biguint(2048);
+    let e = rng.gen_biguint(2048);
+    let m =
```

Comment:
The modulus used here and in the tests is the prime from the [2048-bit MODP DH group](https://tools.ietf.org/html/rfc3526#section-3) (used because I was testing this against I2P's DH agreement).

Reply:
That info belongs in a direct code comment!  And perhaps it will be clearer to parse it from a string in the same grouping as given in that source link, e.g. `"0xFFFFFFFF_FFFFFFFF_C90FDAA2_2168C234_..."`.

(Hmm, do we not parse `_` in bigints?  We should!)

Reply:
(It seems primitive integers don't parse `_` either.  I'm surprised, since it is allowed in integer literals.)